src/scales/scale.time.js

```diff
@@ -562,6 +562,37 @@ module.exports = function(Chart) {
 			me._minorFormat = formats[unit];
 			me._majorFormat = formats[majorUnit];
 			me._table = buildLookupTable(me._timestamps.data, min, max, options.distribution);
+			me._leftOffset = 0;
+			me._rightOffset = 0;
+
+			if (options.offset && ticks.length) {
```

Comment:
Referring to this case (none linear data with linear ticks):

![image](https://user-images.githubusercontent.com/3874900/29162856-f0b266a2-7dba-11e7-9299-36c6e676f739.png)